wavefront/server/apps/call_processing/call_processing/services/pipecat_service.py

```
@@ -510,7 +511,7 @@ async def run_conversation(
         @transport.event_handler('on_client_connected')
         async def on_client_connected(transport, client):
```

Ruff (0.15.0)

[warning] Unused function argument: `transport`

(ARG001)